gradle/build.Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM eclipse-temurin:11-jdk
```

Comment:
What's this file for?  I don't see any reference to it elsewhere.

Reply:
It's the `tmio/besu-build:1.0` image definition